packages/manager/src/features/PlacementGroups/PlacementGroupsLanding/PlacementGroupsRow.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { InlineMenuAction } from 'src/components/InlineMenuAction/InlineMenuAction';
+import { List } from 'src/components/List';
+import { ListItem } from 'src/components/ListItem';
+import { TableCell } from 'src/components/TableCell';
+import { TableRow } from 'src/components/TableRow';
+import { TextTooltip } from 'src/components/TextTooltip';
+import { Typography } from 'src/components/Typography';
+import { useLinodesQuery } from 'src/queries/linodes/linodes';
+import { useRegionsQuery } from 'src/queries/regions';
+
+import { StyledWarningIcon } from './PlacementGroupsRow.styles';
+
+import { MAX_NUMBER_OF_VMS_PER_PLACEMENT_GROUP } from '../constants';
+
+import type { PlacementGroup } from '@linode/api-v4';
+import type { Action } from 'src/components/ActionMenu/ActionMenu';
+import { Hidden } from '@mui/material';
+
+interface PlacementGroupsRowProps {
+  handleDeletePlacementGroup: () => void;
+  handleRenamePlacementGroup: () => void;
+  placementGroup: PlacementGroup;
+}
+
+export const PlacementGroupsRow = React.memo(
+  ({
+    handleDeletePlacementGroup,
+    handleRenamePlacementGroup,
+    placementGroup,
+  }: PlacementGroupsRowProps) => {
+    const { affinity_type, compliant, id, label } = placementGroup;
+    const { data: regions } = useRegionsQuery();
+    const { data: linodes } = useLinodesQuery();
+    const regionLabel =
+      regions?.find((region) => region.id === placementGroup.region)?.label ??
+      '';
+    const numberOfAssignedLinodesAsString =
+      placementGroup.linode_ids.length?.toString() ?? '';
```

Comment:
Optional chain not needed

```suggestion
    const numberOfAssignedLinodesAsString = placementGroup.linode_ids.length.toString();
```

Reply:
dome as well